rj_gameplay/rj_gameplay/play/penalty_offense.py

```diff
@@ -62,8 +62,9 @@ def tick(
         if self._state == State.INIT:
             self.prioritized_tactics.append(prep_move.PrepMove(world_state))
             num_liners = len(world_state.our_visible_robots) - 1
-            start_pt = np.array([-2.0, 1.5])
-            end_pt = np.array([2.0, 1.5])
+            # 7.5 metres to opponent's goal (6 metres to penalty line, 1 metre behind line, then spare distance to be safe)
```

Comment:
```suggestion
            # 7.5 meters to opponent's goal (6 meters to penalty line, 1 meter behind line, then spare distance to be safe)
```
I know it's another way to spell meter, but everything else in the codebase is spelled the American way lmao